BridgeAppSDK/SBAProfileItem.swift

```diff
@@ -708,6 +709,366 @@ open class SBAStudyParticipantCustomAttributesProfileItem: SBAProfileItemBase {
     }
 }
 
+open class SBAWhatAndWhen: NSObject, Comparable {
+    static var valueKey: String { return #keyPath(value) }
+    static var dateKey: String { return #keyPath(date) }
+    static var isNewKey: String { return #keyPath(isNew) }
+    open var value: SBBJSONValue
+    open var date: NSDate
+    var isNew: Bool
```

Comment:
The isNew flag indicates it's a new data point for this item in the local keychain cache, which gets cleared when it gets added to an appropriate SBBScheduledActivity.